virttest/utils_v2v.py

```diff
@@ -71,9 +71,12 @@ def _get_esx_uri(self):
         """
         Return esx uri.
         """
-        uri = "vpx://root@%s/%s/%s/?no_verify=1" % (self.host,
-                                                    self.vpx_dc,
-                                                    self.esx_ip)
+        if self.vpx_dc and self.esx_ip:
+            uri = "vpx://root@%s/%s/%s/?no_verify=1" % (self.host,
+                                                        self.vpx_dc,
+                                                        self.esx_ip)
+        if not self.vpx_dc and self.esx_ip:
+            uri = "esx://root@%s/?no_verify=1" % self.esx_ip
```

Comment:
if self.esx_ip not match, uri will never initialized

Reply:
@chunfuwen  Thanks for your review.
If that happens, we will get a wrong v2v command, and then it will fail.  so It's not necessary to do some checking here.

Reply:
Done